UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -175,6 +175,7 @@ public void ChangeSprite(int SubCataloguePage, bool Network = true)
 		{
 			isPaletteSet = false;
 			PresentSpriteSet = NewspriteDataSO;
+			cataloguePage = SubCatalogue.FindIndex(SO => SO == NewspriteDataSO);
```

Comment:
what does this do?

Reply:
so, just noting this won't Network it as if it was a catalogue change, it would network it like Some are specified a So for This Sprite handle

Reply:
Yep so it was to fix an issue when you manually set SO (outside of catalogue or not) couldn't go back to the previous catalogue because the cataloguePage variable wasn't updated, so it already thought it was on that SO

Reply:
yeah is ok, just add a note to note it needs to be changed to a catalogue message at some point to optimise it